tests/gradlecheck/FetchPostMergeFailedTestClassTest.groovy

```diff
@@ -88,16 +110,16 @@ class FetchPostMergeFailedTestClassTest {
                 ]
         ]).replace('"', '\\"')
 
-        def result = fetchPostMergeFailedTestClass.getQuery()
+        def result = fetchPostMergeFailedTestClass.getQuery("15d")
 
         assert result == expectedOutput
     }
 
     @Test
     void testGetPostMergeFailedTestClassReturnsKeys() {
         def expectedOutput = ["testClass1", "testClass2"]
-
-        def result = fetchPostMergeFailedTestClass.getPostMergeFailedTestClass()
+        def timeFrame = "15d"
```

Comment:
Should this be taken as an arg?

Reply:
Already its part of the arg sayali using `timeFrame`, this file `FetchPostMergeFailedTestClassTest` is the test cases for generating the query.